### DISCUSSION TITLE
New Up & Coming Google Forms Locked mode bypass + GPT auto answer

### DISCUSSION THREAD

**Discussion by QuackersZeDucko** [3kh0/ext-remover#1010] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Hello everyone, today I am showcasing my new up and coming project script for Google forms which currently features a 60-80% accuracy for google prompts/answers(Im hoping to achieve 90% to maybe even 100% using gpt 4)+  many more (Free and Paid) projects to come. 

Features?
It can currently answer Short answers, Paragraphs, Multiple choice and Checkboxes!

Dropdowns are currently being worked on.

Possible prices: Video.

Is this a monthly thing? NO! It's most likely gonna be like 5$ and there are gonna be plenty of giveaways and 20+ people are getting alpha/beta keys to test before release and they get a free role + key on release.

Discord server: https://discord.gg/qEY9HdCUfa

*Looking for other possible developers and currently hiring staff.*

1st video showcase: https://www.youtube.com/watch?v=S6OyR5Y7tNk (Yea its horrible lol

Thanks for reading have a quackatistic day :)

Note: IF YOU DID NOT GET THE MESSAGE IT IS *NOT* OUT YET! Have good day :)

**Comment by rwg100** [comment]
dagger ziggity zawg you already showcased this and everyone bashed on you for it

**Reply by QuackersZeDucko** [reply]
I don't care abt your guys opinions lol

**Reply by QuackersZeDucko** [reply]
If you don't care then don't

**Reply by v4pl** [reply]
not going to lie the thing is pretty cool just don't get caught

**Reply by v4pl** [reply]
but the price idk?
